tools/topology/topology2/platform/intel/sdw-amp-generic.conf

```diff
@@ -497,13 +497,37 @@ IncludeByKey.NUM_SDW_AMP_LINKS {
 		# to show the existence of aggregation in the topology graph.
 		IncludeByKey.PASSTHROUGH {
 		"false" {
-			Object.Base.route [
-				{
-					source  "gain.21.1"
-					sink    "virtual.sdw-amp"
+			IncludeByKey.SDW_SPK_ENHANCED_PLAYBACK {
```

Comment:
Not sure why this is needed. It is just a virtual widget. So, I guess it will work if it always connects to the gain widget.